about.html

```diff
@@ -0,0 +1,123 @@
+                          </a>
+                    </div>
+                    <div class="d-flex gap-5 justify-content-between align-items-center p-1">
+                    <div class="hamburger d-sm-flex flex-column d-md-none d-lg-none p-3">
+                        <span class="d-block bar"></span>
+                        <span class="d-block bar"></span>
+                        <span class="d-block bar"></span>
+                    </div>
+                    <div class="d-block d-md-none d-lg-none ms-5"> 
+                        <a href="index.html">
+                        <img width="140px" src="./imgs/logo.jpg" alt="Techno logo" class="bg-light" />
+                        </a>
+                  </div>
+                  </div>
+                    <div class="d-none d-md-block me-5">
+                        <ul class="d-flex  justify-content-end align-items-baseline p-1 gap-5 justify-content-between me-5">
+                            <li class="list-inline "><a href="#" class="text-decoration-none nav-link ">About</a></li>
+                            <li class="list-inline "><a href="#" class="text-decoration-none nav-link">Program</a></li>
+                            <li class="list-inline "><a href="#" class="text-decoration-none nav-link">Join</a></li>
+                            <li class="list-inline "><a href="#" class="text-decoration-none nav-link">Sponser</a></li>
+                            <li class="list-inline "><a href="#" class="text-decoration-none nav-link">News</a></li>
+                            <a href="#" class="p-2 list-inline text-decoration-none border border-5 border-danger"><b class="bg-black text-muted align-content-center p-1">Buy Ticket</b></a>
+                        </ul>
+                    </div>
+                </navbar>
+            </section>
+                
+                <ul class="menu">
+                    <li class="nav-item"><a href="about.html" class="nav-link ">About</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">Sponser</a></li>
+                    <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                </ul>
+    
+
+        <section class="about-headline d-flex flex-column gap-3 justify-content-between  align-items-center text-center  p-5 m-auto">
+            <div><h3 class=" h-grd">CREATIVE COMNMONS<br>GLOBAL EVENT 2023</h3></div>
+            <p class="about-para p-font border border-3 border-light p-3 bg-white">This Event will show us that the next big thing isn’t a new technology, it’s a new idea.
+                 Human Security and technology<br> in all lief aspect is that idea. The Human Security concept also includes access to healthcare, <br>economic security, environmental 
```

Comment:
- [ ] In your about page in the mobile version I think it would be great if you give the heading the right font size and font weight here to not let the text like this it should have more width so not be under each other in this way so kindly update it to match the given design and please keep the small heading above the heading like the given design 🤩💯🔥

- Your design
![image](https://user-images.githubusercontent.com/58553711/227085167-4a911369-821c-4326-9824-51d284106f5c.png)
- Given
![image](https://user-images.githubusercontent.com/58553711/227085027-6ed25065-b89d-4f68-9b9b-8cd0e157ec89.png)